src/explore-education-statistics-frontend/src/modules/find-statistics/components/SearchForm.module.scss

```diff
@@ -0,0 +1,45 @@
+@import '~govuk-frontend/govuk/base';
+
+.button {
+  background: $govuk-brand-colour;
+  border: 2px solid govuk-colour('black');
+  border-left: 0;
```

Comment:
This border is a little inconsistent with [GOV.UK](https://www.gov.uk/):

![image](https://user-images.githubusercontent.com/9917868/205059159-cc1aaa38-29fc-4d88-9096-4609635edeb5.png)

Would suggest we remove the border? The current implementation is also a little inconsistent as we don't typically have borders around buttons anyway.